php-practice/section3.php

```diff
@@ -125,10 +125,14 @@
  * HINT: It may help to use paper and pencil and write down what the array contains after each iteration of the loop
  */
 
-while ($REPLACE_THIS_VARIABLE_WITH_YOUR_CONDITION /* Edit the code inside these parentheses */) {
-  array_pop($usersToSendEmail);
+while (!empty($originalArray)) {
```

Comment:
To answer your question about how to fix this loop...

**HINT**: Has `$originalArray` been defined yet? Do we care about this variable?